chia/server/address_manager_store.py

```diff
@@ -97,52 +78,105 @@ async def create_address_manager(cls, peers_file_path: Path) -> AddressManager:
         return address_manager
 
     @classmethod
-    async def serialize(cls, address_manager: AddressManager, peers_file_path: Path) -> None:
-        """
-        Serialize the address manager's peer data to a file.
-        """
-        metadata: list[tuple[str, str]] = []
-        nodes: list[tuple[int, ExtendedPeerInfo]] = []
-        new_table_entries: list[tuple[int, int]] = []
+    async def serialize_bytes(cls, address_manager: AddressManager, peers_file_path: Path) -> None:
```

Comment:
this function doesn't need to bye `async` anymore

Reply:
Done